libs/fgdbg/src/Session.cpp

```diff
@@ -16,25 +16,39 @@
 
 #include "fgdbg/Session.h"
 #include <sstream>
+#include <set>
+#include <iostream>
 
 namespace filament::fgdbg {
 
 void Session::addPasses(const std::vector<Pass>& passes) {
-    this->passes.insert(this->passes.end(), passes.begin(), passes.end());
+    for (auto& pass: passes) { addPass(pass); }
```

Comment:
put the content of the `for` on its own line, which is helps a lot when debugging (e.g. to set a breakpoint).